src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -50,13 +50,15 @@ public async Task CreateCdkDeployment(OrchestratorSession session, CloudApplicat
             _interactiveService.LogMessageLine("Starting deployment of CDK Project");
 
             // Ensure region is bootstrapped
-            await _commandLineWrapper.Run($"npx cdk bootstrap aws://{session.AWSAccountId}/{session.AWSRegion}");
+            await _commandLineWrapper.Run($"npx cdk bootstrap aws://{session.AWSAccountId}/{session.AWSRegion}",
+                needAwsCredentials: false);
 
             // Handover to CDK command line tool
             // Use a CDK Context parameter to specify the settings file that has been serialized.
             await _commandLineWrapper.Run( $"npx cdk deploy --require-approval never -c {Constants.CloudFormationIdentifier.SETTINGS_PATH_CDK_CONTEXT_PARAMETER}=\"{appSettingsFilePath}\"",
                 workingDirectory: cdkProjectPath,
-                environmentVariables: environmentVariables);
+                environmentVariables: environmentVariables,
+                needAwsCredentials: false);
```

Comment:
When doing CDK deploy is when we should pass in AWS credentials. We need to be sure the profile that has been configured is what credentials are using.

I assume this worked for you because you have a default profile which is just being by CDK deploy in your case because no credentials are passed. Please test this feature without having a default profile configured on your system.

Reply:
Fixed this and tested my changes by removing the default profile.